src/transactions/transferFeedUtils.ts

```diff
@@ -111,7 +97,10 @@ export function useTransferFeedDetails(transfer: TokenTransfer, isJumpstart: boo
         subtitle = t('feedItemJumpstartSentSubtitle')
       } else {
         title = t('feedItemSentTitle', { displayName })
-        subtitle = t('feedItemSentInfo', { context: !comment ? 'noComment' : null, comment })
+        subtitle = t('feedItemSentInfo', {
+          context: !subtitleContent ? 'noComment' : null,
```

Comment:
This is the only lingering reference to "comments" ever existing in the codebase, AFAICT. I kept it in since i18n uses it as context to suffix the i18n keys; wasn't sure if we could safely change the key names.

Reply:
ah yes, we could rename it to `feedItemSentInfoV1_93` (but we still need to keep the old key name for backward compat.